src/timepicker/timepicker.component.ts

```diff
@@ -65,11 +73,28 @@ export class TimePicker {
 
 	@Output() valueChange: EventEmitter<string> = new EventEmitter();
 
+	writeValue(obj: any): void {
```

Comment:
We should be able to type this more explicitly:

```suggestion
	writeValue(value: string) {
```